doc/api/deprecations.md

```diff
@@ -3556,7 +3556,7 @@ changes:
 
 Type: Runtime
 
-Calling [`util.promisify`][] on a function that returns a <Promise> will ignore
+Calling [`util.promisify`][] on a function that returns a `Promise` will ignore
```

Comment:
I think the intent was to put a link
```suggestion
Calling [`util.promisify`][] on a function that returns a {Promise} will ignore
```

Reply:
There is another occurrence of just `` `Promise` `` for DEP0132 so I think the original change is more consistent.